tasks/test_add_picklist_values.py

```diff
@@ -0,0 +1,412 @@
+import unittest
+from tasks import add_picklist_values
+from unittest import mock
+import responses
+from cumulusci.salesforce_api.tests.metadata_test_strings import deploy_result
+from cumulusci.tasks.salesforce.tests.util import create_task
+
+# to-do: add otherlast test? assert sorted?
```

Comment:
Added this as a to-do... there are certainly more unit tests that could be added here as the functionality is pretty complex, but I think what I have right now should cover most scenarios. Not sure if it's worth the effort to continue adding to this or not -- I'll happily do so, but I wanted to get this in for review first.